src/etl/Batch_Ingestion_Script.py

```diff
@@ -0,0 +1,118 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC ### Water Quality Batch Ingestion
+
+# COMMAND ----------
+
+import requests       
+import json                             
+from pyspark.sql import Row
+from pyspark.sql.types import *
+import os
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC #### Connect to Landing Zone (Bronze)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC #### Fetch Data From Meersens URL
+
+# COMMAND ----------
+
+url = "https://api.meersens.com/environment/public/water/current"
+api_key = os.getenv("OPENWEATHER_API_KEY")
+
+#partition by city or coordinates to enable parallel processing in spark
+def fetch_data_for_partition(iterator):
+    results = []    #store results from API
+    headers = {"api_key": api_key}
+    for row in iterator:
+        response = requests.get(
+            url,
+            headers=headers,
+            params = {
+                "lat": row.lat,
+                "lng": row.lng
+            }
+        )
+        #store response of json data if successful connection
+        #water_quality_data stores the fetched json as a string for each city
+        if response.status_code == 200:
+            print("API Call Successful")
+            data = response.json()
+            print(json.dumps(data))
```

Comment:
Parse Data as it comes in to fetch desired values, if the response returns empty values , handle it with a error/custom message.